Code/GraphMol/DistGeomHelpers/Embedder.cpp

```diff
@@ -1063,6 +1063,12 @@ void EmbedMultipleConfs(ROMol &mol, INT_VECT &res, unsigned int numConfs,
         "torsion-angle preferences (ETversion) supported");
   }
 
+  if (MolOps::needsHs(mol)) {
+    BOOST_LOG(rdWarningLog)
+        << "Molecule does not have explicit Hs. Consider calling AddHs()"
```

Comment:
Given that the check fails also if only some explicit hydrogens are missing, the error message could be reworded from
`Molecule does not have explicit Hs` to `Some or all explicit Hs are missing`. There are multiple occurrences so you might want to to a mass replacement. If you don't care, I don't particularly care either - just an idea.

Reply:
Agreed that it may happen that some Hs are present, but I think that's not going to be the common situation and it does make the error message a bit less clear.
I'd suggest we leave it as is and change it if ends up looking like people are getting confused.